assets/scripts/main.js

```diff
@@ -5,14 +5,23 @@
    * If the page takes time to be loaded, the user could click multiple
    * times on the place order button and this would generate multiple orders.
    * To prevent this we stop form submit event propagation after the first click.
+   *
+   * Covers the case of woocommerce-german-market second checkout page being
+   * disabled. If form fields validation fails, the order submit button is
+   * enabled again to allow the user to resend the form.
```

Comment:
This addition is more confusing than clarifying now, as both cases are covered with the same code.  Please add any missing info into the first paragraph instead.

Reply:
I agree with Daniel. I do not think we should change the comment at all but, if it needs some improvments, should have been done in the first paragraph.

Reply:
I agree with Daniel. I think the comment was pretty claer and do not need modification at all. If you want to add something, I'd do this in the first paragraph.